security_monkey/watchers/config_recorder.py

```diff
@@ -0,0 +1,112 @@
+#     Copyright 2016 Bridgewater Associates
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+"""
+.. module: security_monkey.watchers.config_recorder
+    :platform: Unix
+
+.. version:: $$VERSION$$
+.. moduleauthor:: Bridgewater OSS <opensource@bwater.com>
+
+
+"""
+from security_monkey.watcher import Watcher
+from security_monkey.watcher import ChangeItem
+from security_monkey.constants import TROUBLE_REGIONS
+from security_monkey.exceptions import BotoConnectionIssue
+from security_monkey import app
+from boto.ec2 import regions
+
+
+class ConfigRecorder(Watcher):
+    index = 'configrecorder'
+    i_am_singular = 'Config Recorder'
+    i_am_plural = 'Config Recorders'
+    # TODO: Replace hardcoded region with `get_available_regions` after next boto3 release
+    # Specific PR here: https://github.com/boto/boto3/pull/531/files
```

Comment:
boto3/531 has been merged. Do we know if it's been included in a release? (Can this TODO go away yet?)